src/pages/usage/getting-started.md

```diff
@@ -41,9 +45,9 @@ curl -k -X POST \
     --data-binary @wh_acd.ibm_clinical_insights_v1.0.zip
 ```
 
-Use POST to create it the first time, or PUT to update it later.
+**NOTE:** Use POST to create it the first time, or PUT to update it later.
 
-The example shows the default `X-Watson-UserInfo` header to specify the supertenant tenant id required to deploy or change these ACD provided cartridges to the shared tenant configuration location. This is the default header and format used by ACD if no Tenant header (tenantHeader) value was set in the ACD deployment to support tenant isolation use that header and remove the bluemix-instance-id prefix.  If you set a Tenant header value on the ACD instance you will use that header instead.  For example, if using the OAuth proxy header with tenancy:
+The command above uses the default `X-Watson-UserInfo` header to specify the supertenant tenant id required to deploy or change these provided cartridges to the shared tenant configuration location. This is the default header and format used by ACD if no Tenant header (tenantHeader) value was set in the ACD deployment to support tenant isolation.   If you set a Tenant header value on the ACD instance you will use that header instead.  Using the OAuth proxy and the mulitenancy with the X-Forwarded-User as the tenantHeader you'd use this command instead with a different header.
```

Comment:
add "path" for the multitenancy path with the ... , or drop the word "the" in front of multitenancy